QgisModelBaker/gui/panel/pg_config_panel.py

```diff
@@ -152,3 +280,112 @@ def is_valid(self):
             result = True
 
         return result, message
+
+    def _pg_service_combo_box_changed(self):
+
+        service = self.pg_service_combo_box.currentData()
```

Comment:
maybe return when `service == self._current_service`?

Reply:
the first time running `self.pg_service_combo_box` is `None` and maybe also the `currentData()` is `None`. In that case the underlying code should still be executed